core/templates/filters/string-utility-filters/normalize-whitespace-punctuation-and-case.pipe.ts

```diff
@@ -50,6 +54,18 @@ implements PipeTransform {
                                 isAlphanumeric(inputLine.charAt(j + 1))) {
               result += currentChar;
             }
+            else
```

Comment:
Put the else on the previous line.
` } else {`. Same for the other else if below.

Reply:
Done! @srijanreddy98